components/footer/footer.scss

```diff
@@ -22,45 +22,58 @@
       justify-content: space-between;
     }
 
-    p {
-      margin-top: 0;
-      margin-bottom: $base;
-    }
-
     &-col {
       display: flex;
       flex-direction: column;
-      color: $secondary;
       line-height: 20px;
 
       .links {
-        margin-bottom: $base;
+        display: flex;
+
+        @include xsm {
+          flex-direction: column;
+        }
+
+        @include sm {
```

Comment:
might be worth changing this and the other `sm` breakpoints to `md` since it looks a bit broken now between ~600-650px